### PR TITLE
Send student question functionality

### LINKED ISSUE #228: Spiąć funkcjonalności związane z zadawaniem pytań wykładowcy
Spiąć ficzury:
- wysyłanie pytania przez studenta
- odebranie go po stronie wykładowcy

### LINKED ISSUE #274: Fix adding student questions
Students questions are added four times - kinda sus

### PR DESCRIPTION
**Pull request by rozchlastywacz**
sending questions works properly, but are added four times instead of one

closes #228 if #274 fixed

### CODE REVIEW

**Review by Petros9 — APPROVED** [review]

File: config-overrides.js [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
-src_dir = process.argv[2];
-build_dir = process.argv[3];
-public_dir = "public";
+let cmd = getFileName(process.argv[1]);
+let src_dir = process.argv[2];
+let build_dir = process.argv[3];
+let public_dir = "public";
 
 console.log("cmd:", cmd);
 console.log("src dir:", src_dir);
```
Comment (Derstilon) [inline config-overrides.js:10]: Is this important?

Comment (rozchlastywacz) [reply config-overrides.js:10]: @ostatni5

Comment (michal367) [reply config-overrides.js:10]: yes

**Review by Derstilon — CHANGES_REQUESTED** [review]
Please remove mock data if we won't use it anymore or at least fix the typo in the name
[mockData](https://github.com/michal367/OI-Project/blob/feature/connect-question-sending/src/lecturer/util/mockData.ts)
This file is getting too big and there is a lot of unused data.

**Review by Petros9 — APPROVED** [review]

**Review by Derstilon — APPROVED** [review]

**Review by michal367 — APPROVED** [review]

**Pull request merged**